mock/py/mockbuild/util.py

```diff
@@ -32,7 +32,7 @@
 from textwrap import dedent
 import time
 import uuid
-
+import jinja2
```

Comment:
There have to be `Requires: python3-jinja` in spec file. And %if condition for EL7 with python2.